packages/render-fidelity-tools/src/common.ts

```diff
@@ -104,7 +104,7 @@ export interface ImageComparisonConfig {
 }
 
 export interface ComparableImage {
-  [index: number]: number;
+  [index: number]: number;  // what does this syntax mean?
```

Comment:
This was fine as an excuse for a first PR, but is probably not the most efficient means of getting your questions answered about the code. I'd go ahead and revert this last commit and then we can merge this. Let's do a screen share and talk through code questions you have.